snapshots/manager.go

```diff
@@ -59,12 +45,39 @@ type Manager struct {
 	restoreChunkIndex  uint32
 }
 
+// operation represents a Manager operation. Only one operation can be in progress at a time.
```

Comment:
Moved constants to be below type definitions

Reply:
Nice, thank you @p0mvn!